browser/css/color-palette.css

```diff
@@ -12,5 +12,8 @@
 	--gray-color: #b6b6b6;
 	--white-bg-color: #fff;
 
+	--color-toolbar: #fff;
```

Comment:
I think it makes total sense to have `--color-border` but I would really avoid starting to have `--color-component-x` because that, I'm afraid, will lead to multiple variables with the same value but different naming 

In terms of naming conventions, I think better would be to have anything between these categories:
- Main colors:
  -  Primary, (and if needed secondary, branding etc)
- Error | Success  | Warning
- Attribute:
  - Text
  - Background
  - Border
  - Radius
  - Shadow

Reply:
That's fine.  

So in this case `--color-toolbar` will be `--color-background-primary` ? cause `--color-background` will be the color of the document and the tabbs?

Reply:
What is with select, hover. Should they be part of the name convention? `--color-select` and `--color-hover` ?

Reply:
I use `--color-main-background` which was used also at NC for #fff

Reply:
thanks, merging. Ideally we would have something like --color-secondary or --color-secondary-primary-element